src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/fingerprints/ShortsPlayerConstructorFingerprint.kt

```diff
@@ -0,0 +1,35 @@
+package app.revanced.patches.youtube.layout.sponsorblock.bytecode.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.youtube.layout.sponsorblock.annotations.SponsorBlockCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+@Name("shortsplayer-constructor-fingerprint")
+@MatchingMethod("Lhgp;", "<init>")
+@SponsorBlockCompatibility
+@Version("0.0.1")
+object ShortsPlayerConstructorFingerprint : MethodFingerprint(
+    "V", AccessFlags.PUBLIC or AccessFlags.CONSTRUCTOR, listOf("L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "L", "[B", "[B", "[B", "[B", "[B"), listOf(
```

Comment:
Is it possible to get rid of the parameters list? In my experience the build process affects those quite often, maybe the opcode pattern can be adjusted or a different parent fingerprint can be found.

Reply:
1) I don't know if I remove pattern lists without consequences. I'll try.
2) A different fingerprint (bytecode source location) can't be found, because this is already the better one.

Reply:
Ok...I was able to completely remove the method pattern list, and leaving just the opcodes.